index.html

```diff
@@ -0,0 +1,162 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Aladdin</title>
+</head>
+
+<body>
+    <div class="container"> 
+    <div class="header">
+        <nav>
+            <ul>
+                <li><h2>Aladdin</h2></li>
+                <li><button class="fas fa-bars hamburger-icon menu"></button></li>
+            </ul>
+        </nav>
+    </div>
+
+    <main>
+        <div class="main">
+            <div class="headline">
+                <h1>Hey There. I am Aladdin</h1>
+                <h3>I am a Software Developer</h3>
+                <p>I can help you build a product, feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, do not hesitate to contact me.</p>
+            </div>
+
+            <div class="vector">
+                <ul>
+                    <li><a href="#"><i class="fab fa-github"></i></a></li>
+                    <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+                </ul>
+            </div>
+
+        </div>
+
+        <!-- toolbar-->
+        
+        <div class="grid_container"><!-- section one-->
+            <div>
+                <h1 id="tool_head">My Recent Works</h1>
+                <img id="tool-img" src="Images/Placeholder.png" alt="first_project_image">
+                <h2 id="tool-sub-header">Multi-Post Stories</h2>
+                <p id="tool-prgf">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    Has been the industry's standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                </p>
+            </div>
+            <div>
+                <ul id="tool-btns">
+                    <li class="tool-btn">css</li>
+                    <li class="tool-btn">html</li>
+                    <li class="tool-btn">bootstrab</li>
+                    <li class=tool-btn>ruby</li>
```

Comment:
- [ ] The list of icons also should have a `a` tags wrapped into see requirement 👇
![icon](https://user-images.githubusercontent.com/61562626/129426227-961b9e1b-9645-40c1-8e0d-97b5699a29de.jpeg)